http_service/bugbug_http/app.py

```diff
@@ -115,13 +170,12 @@ def schedule_bug_classification(model_name, bug_ids):
     job_id_mapping = {get_mapping_key(model_name, bug_id): job_id for bug_id in bug_ids}
     redis_conn.mset(job_id_mapping)
```

Comment:
This part is now in schedule_job and should be removed, right? (note: move the comment too)

Reply:
No, because `schedule_job` sets a mapping key that contains the entire list of bug ids, whereas this is setting mapping keys for each individual bug id. This is needed in case someone calls:
`/model/predict/batch` (while posting bug 123456)

and then later someone calls:
`/model/predict/123456`

This way a mapping key for the second call already exists and we don't do unnecessary work.

Reply:
Oh, OK. I wonder if that could be a problem, is there a limit on the length of the key?

Reply:
Apparently the redis key size limit is 512MB.. so we're probably good :)

That said, we don't actually use the key that has all the bug ids in it.. we only use that to schedule the job, and then the worker adds results for each individual bug. So we could potentially not set that key if you are worried (though doing so makes the two endpoints diverge more in implementation).